scripts/get

```diff
@@ -75,10 +75,26 @@ latest_version() {
 
     if [[ $VERSION == "" ]]; then
         echo No version provided, finding latest successful GitHub actions run
-        VERSION=$(curl -sSfL "${GH_HEADERS[@]}" "$GH_API_URL" |
-            jq -r '.workflow_runs | map(select(.name == "test" and .head_branch == "master" and .conclusion == "success")) | first | .head_sha')
 
-        echo "Using latest successful GitHub action master: $VERSION"
+        for ((i = 1; i < 10; i++)); do
```

Comment:
I have never used GH API, but I was just wondering if they allow us to fetch pages by specifying size? Then how about fetching just one page with large enough page size. That way we avoid the pagination implementation at our side. 

Sorry if this sounds silly, I am just thinking out loud.

Reply:
Yeah this is cool, this way we don't have to hard code num pages to iterate on.